### PR TITLE
[Proposal] Artisan command for one click installation services

### PR DESCRIPTION
**Pull request by joecohens**
This is a proposal to this issue https://github.com/cachethq/Cachet/pull/96

### CODE REVIEW

**Comment by joecohens** [comment]
@GrahamCampbell Omg I've tried to fix all StyleCI problems, I think is time to install it locally to test before push haha :)

**Comment by jbrooksuk** [comment]
Also, what are we doing with #96?

**Comment by GrahamCampbell** [comment]
Yeh, that will be possible in the future, but now right now. Using a custom version of phpcsfixer, and its changing quite frequently. In a few weeks, I'll be able to give you config file  that'll with the 1.5 release when it's done.

**Comment by GrahamCampbell** [comment]
Also, we don't need the double constructor here.

File: app/commands/OneClickDeployCommand.php [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+     * @var string
+     */
+    protected $description = 'Installs CachetHQ on one click supported services';
+
+    /**
+     * Create a new command instance.
+     *
+     * @return void
+     */
+    public function __construct()
```
Comment (GrahamCampbell) [inline app/commands/OneClickDeployCommand.php]: this should be deleted

**Comment by jbrooksuk** [comment]
It's not possible for me to push to this, is it?

**Comment by GrahamCampbell** [comment]
No, you'd need to pull it down to a branch on this repo. I can do that for you if you like.

**Comment by jbrooksuk** [comment]
Please, I've got to go out.

**Comment by GrahamCampbell** [comment]
Actually, I'm going to totally redo this. This hasn't been done correctly.

**Comment by GrahamCampbell** [comment]
After #198 has gone through, I'll rewrite this.

**Comment by joecohens** [comment]
@GrahamCampbell Thanks :)

**Comment by GrahamCampbell** [comment]
Coming quite soon...

**Comment by GrahamCampbell** [comment]
Also, I'm going to make sure we support `--force` like laravel does.

**Comment by jbrooksuk** [comment]
It'd be interesting to see how you do that :)

**Comment by GrahamCampbell** [comment]
and `--database`

**Comment by joecohens** [comment]
Me too!

**Comment by GrahamCampbell** [comment]
It's super simple actually. :)

**Comment by GrahamCampbell** [comment]
And `--seed` too. :)

**Pull request closed without merging**